src/components/facet-search.tsx

```diff
@@ -2,17 +2,15 @@ import {
   FacetSearch as HeadlessFacetSearch,
   FacetSearchState,
 } from '@coveo/headless';
-import {FunctionComponent, useState} from 'react';
+import {FunctionComponent} from 'react';
 
 export interface FacetSearchProps {
   controller: HeadlessFacetSearch;
   facetSearchState: FacetSearchState;
 }
 
 export const FacetSearch: FunctionComponent<FacetSearchProps> = (props) => {
-  const [searchBoxValue, setSearchBoxValue] = useState('');
```

Comment:
don't need a local state, now the query's state is exposed from headless, that can be used instead